examples/csv_read_with_callback_loading/csv_read_with_callback_loading.py

```diff
@@ -14,8 +14,11 @@ class Plugin:
     def read_store(self, parameter):
         file_path = os.path.realpath(parameter["file"])
 
-        with open(file_path, newline="") as csvfile:
+        with open(file_path, newline="", encoding="utf8") as csvfile:
             tab_delimiter = "\t"
+            # skip the first 4 lines
+            # for i in range(4):

```

Comment:
Is it intended to be commented out?

Reply:
yes it should only complement the example and might be useful.